_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -90,63 +84,47 @@ Single autocomplete term for select.
 
 #### Passed Example 3
 
-Autocomplete term, only valid for textarea.
+This `autocomplete` [attribute value][] only has the required token, and is valid for a `textarea` element. Missing upper and lower case letters is allwed for `autocomplete` attributes.
 
 ```html
 <textarea autocomplete="Street-Address"></textarea>
 ```
 
 #### Passed Example 4
 
-Two autocomplete terms.
+This `autocomplete` [attribute value][] has a `work` token, allowed because it is used before `email`. The `email` taken is allowed on `input` elements with a `type` [attribute value][] of `text`.
 
 ```html
 <label>Work email<input autocomplete="Work Email"/></label>
 ```
 
 #### Passed Example 5
 
-Autocomplete using section-\*
+This `autocomplete` [attribute value][] has a `section-` token, which can preface any [correct autocomplete field][]. The `email` taken is allowed on `input` elements with a `type` [attribute value][] of `text`.
 
 ```html
 <label>Email<input autocomplete="section-partner email"/></label>
 ```
 
 #### Passed Example 6
 
-Triple autocomplete terms.
+This `autocomplete` [attribute value][] has `section-`  and `billing` tokens. These tokens can preface any [correct autocomplete field][]. The `email` taken is allowed on `input` elements with a `type` [attribute value][] of `text`.
 
 ```html
 <label>Address<input type="text" autocomplete="section-primary billing address-line1"/></label>
 ```
 
 #### Passed Example 7
 
-Full length autocomplete terms.
+This `autocomplete` [attribute value][] has all allowed types of tokens in the correct order. The `email` taken is allowed on `input` elements with a `type` [attribute value][] of `text`.
 
 ```html
 <label>Email<input autocomplete="section-primary shipping work email"/></label>
 ```
 
 #### Passed Example 8
 
-This `input` element has an [explicit role][] of `none`. However, it is [focusable][] (by default). Thus it has a [semantic role][] of `textbox` due to [Presentational Roles Conflict Resolution][]. It has a single autocomplete term.
-
-```html
-<label>Username<input role="none" autocomplete="username"/></label>
-```
-
-#### Passed Example 9
-
-The `input` element does not participates in sequential focus navigation, but still has a semantic role that is a widget role, and has a single autocomplete term.
-
-```html
-<label>Username<input tabindex="-1" autocomplete="username"/></label>
-```
-
-#### Passed Example 10
-
-The `input` element does not have a semantic role that is a widget role, but still participates in sequential focus navigation because of the [`tabindex` attribute](https://html.spec.whatwg.org/#the-tabindex-attribute), and has a single autocomplete term.
+The `autocomplete` attribute value is on an `input` element that does not have a semantic role that is a widget role, but still participates in sequential focus navigation because of the [`tabindex` attribute](https://html.spec.whatwg.org/#the-tabindex-attribute).
```

Comment:
```suggestion
The `autocomplete` attribute value is on an `input` element that does not have a semantic role that is a widget role, but still participates in [sequential focus navigation][] because of the [`tabindex` attribute](https://html.spec.whatwg.org/#the-tabindex-attribute).
```
Plus, we usually don't link to specs for elements and attributes names, so the link to `tabindex` could probably be removed.